### DISCUSSION TITLE
solver error while performing calculate_sensitivities

### DISCUSSION THREAD

**Discussion by swhan22** [pybamm-team/PyBaMM#2792] [Q&A]
Hi all,

When I try to compute cell voltage sensitivity w.r.t. active material volume fraction, (e.g., sim["Terminal voltage [V]"].sensitivities["Negative electrode active material volume fraction"], the Casadi solver has no problem computing it.

However, when I try to calculate cell voltage sensitivity w.r.t. either inner or outer SEI thickness (e.g., sim["Terminal voltage [V]"].sensitivities["Initial outer SEI thickness [m]"]), I get the following error:

--> SolverError: Could not find consistent states: Could not find acceptable solution: .../casadi/core/rootfinder.cpp:280: rootfinder process failed. Set 'error_on_fail' option to false to ignore this error.

Would any one know what is causing this issue? Is there any solver setting that I could change to fix the problem? Or could it be that this is because the SEI layer is a part of a submodel? Thanks.

**Comment by Mrzhang-hub** [comment]
In order to better help you, can you provide the necessary code to reproduce your result?
